### ISSUE TITLE
[BE] Created ticket is saved to the Database

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#60]
The created ticket and the corresponding input is saved to the Database.

## Acceptance criteria
- [ ] The correct information is stored
- [ ] is tested

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
